conformance/failure_list_js.txt

```diff
@@ -1,113 +1,13 @@
-Recommended.Proto2.ProtobufInput.ValidDataRepeated.BOOL.PackedInput.DefaultOutput.ProtobufOutput
```

Comment:
I think you should pick the github change for this file

Reply:
Done